index.html

```diff
@@ -6059,7 +6059,7 @@ <h2>Definition of Roles</h2>
               </p>
             </div>
             <h4><abbr title="Mathematical Markup Language">MathML</abbr> Example with Embedded TeX Annotation</h4>
-            <pre class="example highlight">
+            <pre class="example highlight xml">
```

Comment:
I think this one is fine to be html? It's perhaps unimportant though.

Reply:
Like SVG it's an XML language. But I agree it doesn't matter much.